protx-client/src/components/ProTx/components/dashboard/DisplaySelectors.jsx

```diff
@@ -98,6 +100,7 @@ function DisplaySelectors({
   const valueRadioBtn1 =
     mapType === 'maltreatment' ? 'rate_per_100k_under17' : 'count';
   const display = useSelector(state => state.protx.data.display);
+  const analyticsCategories = [{name: 'risk', display_text: 'Relative Risk'}, {name: 'pred_per_100k', display_text: 'Predicted Number of Cases'}]
```

Comment:
reminder for us to ask Juliana about the "Predicted Number of Cases " string.  I wonder if we need the a "(per 100k)" added 🤷